services/account/apps/api/src/services/handles.service.ts

```diff
@@ -17,4 +19,13 @@ export class HandlesService {
     }
     throw new Error('Invalid msaId.');
   }
+
+  async getExpiration(): Promise<number> {
+    const lastFinalizedBlockNumber = await this.blockchainService.getLatestFinalizedBlockNumber();
+    return lastFinalizedBlockNumber + 10;
```

Comment:
FYI the standard expiration we adopted in SIWF for claiming a handle is 10 minutes (= 600 / SECONDS_PER_BLOCK) blocks.

Though I wonder if (time in seconds) should be an (optional) input parameter to the `GET` endpoint; if not provided we could assume 10 minutes. I think we should add that; it's easy enough. If there are no objections @mattheworris @claireolmstead , I'll add it to the issue AC.